Translators/WZDx/main.py

```diff
@@ -6,12 +6,20 @@
 from request_wrapper import get_sdw_request, get_rsu_request
 from tim_generator import generate_tim
 from flask import request, Flask
+from datetime import datetime
 
 app = Flask(__name__)
 
 log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
 logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
 
+def record_active(feature):

```

Comment:
While we're in here should we add a check for end_date being past as well? I don't think we'll see those but might be a good idea to have a check just in case.

Reply:
Added a check to this method that current time is not past end_date